edi_webservice/models/edi_backend.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2020 Creu Blanca
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class EdiBackend(models.Model):
+
+    _inherit = "edi.backend"
+
+    webservice_backend_id = fields.Many2one("webservice.backend")
+
+    def _get_component_usage_candidates(self, exchange_record, key):
```

Comment:
can we think about how this can be simplified by using this new approach? https://github.com/OCA/edi/pull/295

Maybe we can extend the component match to `_webservice_protocol`. No?

If this works for you I'll make that piece a bit more flexible.

Reply:
You are right, but maybe I should need to wait for the approval of the commit, isn't it? We can keep this PR as DRAFT until the merge, but it seems right to me :wink:

Reply:
you are right. Go ahead if this unlocks your work but add a TODO for refactoring.
Look at my comment on the adapter class.